packages/storefront-events-collector/src/handlers/requisitionList/addToRequisitionListAEP.ts

```diff
@@ -0,0 +1,43 @@
+import { Event } from "@adobe/magento-storefront-events-sdk/dist/types/types/events";
+import { sendEvent } from "../../alloy";
+import { BeaconSchema } from "../../types/aep";
+import { createProductListItems } from "../../utils/aep/productListItems";
+
+const XDM_EVENT_TYPE = "commerce.requisitionListAdds";
+
+/** Sends an event to aep with an addToRequisitionList payload */
+const handler = async (event: Event): Promise<void> => {
+    const { changedProductsContext, requisitionListContext, debugContext, customContext, storefrontInstanceContext } =
+        event.eventInfo;
+
+    let payload: BeaconSchema;
+    if (customContext && Object.keys(customContext as BeaconSchema).length !== 0) {
+        // override payload on custom context
+        payload = customContext as BeaconSchema;
+    } else {
+        payload = {
+            commerce: {
+                requisitionList: {
+                    ID: requisitionListContext?.id,
+                    name: requisitionListContext?.name,
+                    description: requisitionListContext?.description,
+                },
+            },
+            productListItems: createProductListItems(changedProductsContext, storefrontInstanceContext),
+        };
+    }
+
+    payload.commerce = payload.commerce || {};
+
+    payload.commerce.requisitionListAdds = {
+        value: 1,
+    };
+
+    payload._id = debugContext?.eventId;
+    payload.eventType = XDM_EVENT_TYPE;
+    console.log(payload);
+    sendEvent(payload);
+    debugger;
```

Comment:
```suggestion
    sendEvent(payload);
```

Reply:
i removed this in my next push. forgot i left that there

Reply:
removed